lib/omniauth/strategies/cas/service_ticket_validator.rb

```diff
@@ -52,7 +52,11 @@ def parse_user_info(node)
               unless e.kind_of?(Nokogiri::XML::Text) || node_name == 'proxies'
                 # There are no child elements
                 if e.element_children.count == 0
-                  hash[node_name] = e.content
+                  hash[node_name] = if hash[node_name] && @options.merge_multivalued_attributes
+                                      Array(hash[node_name]).push(e.content)
+                                    else
+                                      e.content
+                                    end
```

Comment:
This would be better stubbed out in a separate method

Reply:
@vjt Thanks for the feedback, see the updated changes. And glad to see this project is still active. 🎉